### ISSUE TITLE
Support obtaining the path quadruple of one file

### ISSUE THREAD

**Issue by MagpieRYL** [falcosecurity/falco#2613]
**Motivation**

file system isolation breaking is a common attack surface of container escape, which can be easilier detected with the path-quadruple.

**WHT'S path-quadruple :** `fd.name` when a process accesses `/mnt/a/z/d` inside a container, using `read/write/open/etc.`

- path_current_namespace_wrapped: /mnt/a/z/d
- path_current_namespace_clean: /mnt/a/b/c/d   → falco obtains at present ?
- path_host_namespace_wrapped: /a/z/d
- path_host_namespace_clean: /a/b/c/d

```
a/
 +--b/
 |  +--c/
 |     +--d
 +--z: is a symbol-link pointing `a/b/c`

HOST
---------------- `docker run -v /a:/mnt/a`
CONTAINER

/mnt/a/
     +--b/
     |  +--c/
     |     +--d
     +--z
```

👆 with the path-quadruple, we can easily detect the container escaping of the file system breaking, which is caused by the unsafe mount configs and some CVEs.

**Feature**

support parsing path-quadruple of all the file path fields.

**Comment by Andreagit97** [comment]
This sounds like an interesting feature to have, thank you for reporting! Cannot promise any delivery date right now but we can discuss it in the next maintainer calls! cc. @loresuso

**Comment by loresuso** [comment]
Agreed, monitoring file system access is for sure the way to go detect the effects of a container escape. Let's discuss how we can work on this!
